kairon/chat/handlers/channels/clients/whatsapp/gupshup.py

```
@@ -0,0 +1,272 @@
+        headers = {
+            "D360-API-KEY": config.get("api_key")
+        }
+
+        resp = requests.get(
+            endpoint,
+            headers=headers,
+            timeout=10
+        )
+
+        if resp.status_code != 200:
+            raise AppException(
```

Ruff (0.15.15)

[error] Undefined name `AppException`

(F821)